qemu/tests/netperf_stress.py

```diff
@@ -121,6 +127,9 @@ def netperf_test(duration):
     for client in netperf_client:
         c_info = {}
         if client in vms:
+            if params.get("os_type") == "windows":
+                params["start_vm"] = "yes"
+                env_process.preprocess_vm(test, params, env, client)
```

Comment:
I will prefer `start_vm_vm2 = no` instead, that's all we want and no extra process would be requested here.